dist/profile/templates/confluence/vhost.conf

```
@@ -2849,12 +2849,18 @@ RewriteRule "^/display/JENKINS/ZMQ\+Event\+Publisher\+Plugin$" "https://plugins.
 RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
 RewriteRule "^/display/JENKINS/Zoom\+Plugin$" "https://plugins.jenkins.io/zoom" [NC,L,QSA,R=301]
 RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
-RewriteRule "^/display/JENKINS/IBM\+zOS\+Connector$" "https://plugins.jenkins.io/zos-connector" [NC,L,QSA,R=301]
+RewriteRule "^/display/JENKINS/IBM\+zOS\+Connector\+Plugin$" "https://plugins.jenkins.io/zos-connector" [NC,L,QSA,R=301]
+RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
```

The page https://wiki.jenkins.io/display/JENKINS/IBM+zOS+Connector+Plugin does not exist on the Jenkins wiki.

The page https://wiki.jenkins.io/display/JENKINS/IBM+zOS+Connector exists on the wiki and was correctly redirected by the line before this change.  These two lines need to be removed and the original line restored
```suggestion
RewriteRule "^/display/JENKINS/IBM\+zOS\+Connector$" "https://plugins.jenkins.io/zos-connector" [NC,L,QSA,R=301]
```

---

Fixed by #1591